recipes-kernel/linux/5.4/defconfigs/xenclient-syncvm/defconfig

```diff
@@ -33,8 +34,13 @@ CONFIG_X86_CPUID=y
 CONFIG_HZ_100=y
 # CONFIG_RANDOMIZE_BASE is not set
 CONFIG_LEGACY_VSYSCALL_EMULATE=y
-# CONFIG_ACPI is not set
```

Comment:
I assume this is another instance where CONFIG_ACPI is default =y in the Kconfig?

Reply:
Yes, you are correct - `default y if X86`.